src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/TseHelperTest.java

```diff
@@ -220,14 +220,14 @@ void getReplyDocumentRequest_generatesData() throws JsonProcessingException {
         String replyDocumentRequest = TseHelper.getReplyDocumentRequest(caseData, "");
         String expected = "{\"accessKey\":\"\",\"templateName\":\"EM-TRB-EGW-ENG-01212.docx\","
             + "\"outputName\":\"Withdraw my claim Reply.pdf\",\"data\":{\"caseNumber\":\"1234\","
-            + "\"type\":\"Withdraw my claim\",\"supportingYesNo\":\"Yes\","
+            + "\"type\":\"Withdraw my claim\",\"responseDate\":\"16-May-1996\",\"supportingYesNo\":\"Yes\","
             + "\"documentCollection\":[{\"id\":\"1234\","
             + "\"value\":{\"typeOfDocument\":null,"
             + "\"uploadedDocument\":{\"document_binary_url\":\"http://dm-store:8080/documents/1234/binary"
             + "\",\"document_filename\":\"image.png\","
             + "\"document_url\":\"http://dm-store:8080/documents/1234\"},\"ownerDocument\":null,"
             + "\"creationDate\":null,\"shortDescription\":null}}],\"copy\":\"Yes\","
-            + "\"response\":\"response\"}}";
+            + "\"response\":\"response\",\"respondent\":\"Claimant\"}}";
```

Comment:
respondent = Claimant? is this not a bit confusing? Could this be called something else?

Reply:
I understand why it's called respondent in some sense as it's the person respondent to the application but in our business terms, it might be confusing. If others think the same then I would suggest changing it

Reply:
I kind of see what you mean. I renamed the field respondentParty and its values could be Claimant or Respondent as both make TSE response action.

Reply:
Maybe something like responseFrom would've been better but I guess it works. You might also need to update the docmosis template as well